include/vast/Dialect/HighLevel/HighLevelOps.td

```diff
@@ -339,7 +339,7 @@ def HighLevel_FieldDeclOp
 
 def HighLevel_RecordMemberOp
   : HighLevel_Op< "member" >
-  , Arguments<(ins AnyType:$record, FlatSymbolRefAttr:$field)>
+  , Arguments<(ins AnyType:$record, Core_MemberVarSymbolRefAttr:$field)>
```

Comment:
Will this work properly with function pointers inside structs?

Reply:
huh? Core_MemberVarSymbolRefAttr is just alias for FlatSymbolRefAttr.

Reply:
Nevermind, yeah. At first I thought it might break the lookup when a call of a function happens using record member, but it was just a brainfart.